tests/predict/test_class_domain_nets.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import torch
+
+from kale.predict.class_domain_nets import (
+    ClassNetSmallImage,
+    ClassNetVideo,
+    ClassNetVideoConv,
+    DomainNetSmallImage,
+    DomainNetVideo,
+    SoftmaxNet,
+)
+
+BATCH_SIZE = 2
+INPUT_BATCH = torch.randn(BATCH_SIZE, 128)
+INPUT_BATCH_LOGITS = torch.randn(BATCH_SIZE, 1024, 1, 1, 1)
```

Comment:
I could guess what it is for but adding some light comment about what line 15 is will help users or new comers. This code not only serves the purpose of test coverage, but also can teach others how to write proper tests. Thanks.

Reply:
@XianyuanLiu I should make it clearer: could you explain a bit `INPUT_BATCH_LOGITS` vs `INPUT_BATCH`? Is `INPUT_BATCH_LOGITS` some synthetic output from previous layers?

Or provide some references about the `_LOGITS`. It is not as obvious as `INPUT_BATCH`.

Reply:
> @XianyuanLiu I should make it clearer: could you explain a bit `INPUT_BATCH_LOGITS` vs `INPUT_BATCH`? Is `INPUT_BATCH_LOGITS` some synthetic output from previous layers (i.e. output of the I3D)?
> 
> Or provide some references about the `_LOGITS`. It is not as obvious as `INPUT_BATCH`.

Or maybe changing `_LOGITS` to `_I3D`?